assemblies/assembly-about-rhdh.adoc

```diff
@@ -12,11 +12,20 @@ endif::[]
 
 :context: about-rhdh
 
-{product} ({product-very-short}) is an enterprise-grade internal developer portal designed to simplify and streamline software development processes. Combined with {ocp-brand-name}, {product-very-short} empowers platform engineering teams to create customized portals that improve developer productivity, accelerate onboarding, and enable faster application delivery. By reducing friction and complexity, {product-very-short} allows developers to focus on writing high-quality code while adhering to enterprise-class best practices.
+{product} ({product-very-short}) is an enterprise-grade internal developer portal (IDP) that simplifies and accelerates software delivery. It provides a customizable web-based interface that centralizes access to key development resources—including source code repositories, CI/CD pipelines, APIs, documentation, and runtime environments. The platform supports cloud-native environments such as Kubernetes, {ocp-brand-name}, and hybrid infrastructure.
```

Comment:
https://github.com/redhat-documentation/vale-at-red-hat/pull/938 to address the following warning:

[vale] reported by reviewdog 🐶
[RedHat.Slash] Use either 'or' or 'and' in 'CI/CD'